documentation/book/con-topic-operator-cluster-label.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// assembly-deploying-the-topic-operator.adoc
+
+[id='con-topic-operator-cluster-label-{context}']
+
+= Identifying a Kafka cluster for topic creation
```

Comment:
It's not just about "creation" I would say handling in general.
If you create a topic specifying the right label so that the TO creates the topic and then you change the label in something different, the TO isn't able to "watch" it anymore so if you change the resource (i.e. the configuration), these changes don't have effect.

Reply:
I see that we talk about creation even in the coming lines so I would talk about handling and not just creation, wdyt @scholzj ?

Reply:
I agree, we should make this more generic to not talk just about creating.

Reply:
Changed to 'Identifying a Kafka cluster for topic handling'